src/integrationtest/python/JanusGraphContainer.py

```diff
@@ -44,11 +54,14 @@ def pull_image(self):
         image = self.client.images.pull(self.BASE_IMAGE)
         return image
 
+    def get_host_ip(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/JanusGraph/janusgraph-python/pullRequest?prid=4792911)